wiki/en/Installation-for-Linux.md

```diff
@@ -13,26 +13,24 @@ Make sure you read the [Getting Started](Getting-Started) page.
 
 {% include infobox_each_os.html %}
 
-Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.
+**Upgrading?** You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.
 
-### Debian and Ubuntu
-
-We provide three equivalent `.deb` files for the most common architectures. Please download the appropriate one:
+**Installing a server?** Read the [guide for server adminstrators](Running-a-Server) first. 
 
-**For Intel/AMD based machines (amd64):**
+### Debian and Ubuntu
 
 [Download Jamulus (.deb, amd64)]({{ site.download_root_link }}{{ site.download_file_names.deb-gui }}){:.button}
 
-**For ARM based machines (e.g. Raspberry Pi, armhf, arm64):**
+**ARM-based machines** (e.g. Raspberry Pi): [armf]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-armhf }}) or [arm64]({{ site.download_root_link }}{{ site.download_file_names.deb-gui-arm64 }})
```

Comment:
I'd put "download for" or "download .deb" somewhere

Reply:
OK, but I do I think anyone who is choosing to run a Jamulus **Client** on a microcomputer will know what they're doing and just needs to see a link.

Reply:
We're all on microcomputers. ;) Though if I want client on a Pi4 I'll use jambox. Should we mention it here?

Reply:
No, that's my point. If we start listing every exotic Linux-based hardware on which somebody might possibly run Jamulus, then we start confusing or annoying the vast majority of users who just want to run it on a standard machine. Less than 5% of all downloads for Jamulus are for Linux as it is (and most of those are probably server users). We support x64 and (for the minority of knowledgeable people) AMD. Keep it simple.

Reply:
> We support x64 and (for the minority of knowledgeable people) AMD. Keep it simple.

You mean ARM 😄

Reply:
5% are downloading our precompiled binaires... Ok
There are also lots of users using there distribution's repository. For them, non problem with the install.